kubernetes/argocd/stacks/common/vault-values-override.yml

```diff
@@ -80,6 +80,7 @@ server:
       - mountPath: /vault/userconfig/vault-ha-tls
         name: userconfig-vault-ha-tls
         readOnly: true
+  logLevel: debug
```

Comment:
do we want to check this in?

Reply:
You're right. Removed in [4ad7ce4](https://github.com/acm-uic/IaC/pull/58/commits/4ad7ce4c4e8ab98e277e3cece57e1af4b4722f6a).